css/style.css

```diff
@@ -61,18 +61,29 @@ h1 {
 }
 
 .loader-3 {
-    /* Your code goes here */
-    /* Remove this example */
-    width: 32px;
-    height: 32px;
-    border-radius: 50%;
-    border-top: 3px solid rgba(255, 255, 255, 0.5);
-    border-right: 3px solid rgba(255, 255, 255, 0.5);
-    border-bottom: 3px solid rgba(255, 255, 255, 0.5);
-    border-left: 3px solid rgba(255, 255, 255, 1);
-    animation: loader 1s cubic-bezier(.41, .25, .32, .83) infinite;
+    filter: blur(1.3px);
+}
+.loader-3_item-1 {
```

Comment:
I suggest keeping naming consistent (kebab case). e.g.` loader-3-item-1`

Reply:
fixed to kebab-case